mne/viz/utils.py

```diff
@@ -2875,7 +2878,9 @@ def _plot_masked_image(ax, data, times, mask=None, yvals=None,
                        linewidths=[.75], corner_mask=False,
                        antialiased=False, levels=[.5])
         time_lims = times[[0, -1]]
-        ylim = yvals[0], yvals[-1] + 1
+        time_lims = [extent[0], extent[1]]
```

Comment:
forgot to delete the first definition of `time_lims`

Reply:
Corrected.